### ISSUE TITLE
Cannot deprecate snapshot

### ISSUE THREAD

**Issue by effigies** [OpenNeuroOrg/openneuro#2699]
**Describe the bug**
Attempting to deprecate a snapshot fails with no error.

**To Reproduce**
1. Go to https://openneuro.org/datasets/ds004218/versions/1.0.0/deprecate
2. Enter a deprecation reason (I used "This version contains incomplete files")
3. Click button
4. Nothing seems to happen

**Expected behavior**
Snapshot should be deprecated

**Comment by nellh** [comment]
A fix for this is in #2700 but I ran the query to deprecate this snapshot in the meantime.

```graphql
mutation deprecateSnapshot {
  deprecateSnapshot(datasetId: "ds004218", tag: "1.0.0", reason: "This version contains incomplete files") {
    id
  }
}
```

**Comment by effigies** [comment]
Thanks, Nell!
